lib/Server.js

```diff
@@ -444,6 +444,9 @@ Server.prototype.checkHost = function(headers) {
 	// always allow localhost host, for convience
 	if(hostname === "127.0.0.1" || hostname === "localhost") return true;
 
+	// always allow requests with explicit IP-address
+	if(/^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4}$/.test(hostname)) return true;
```

Comment:
I'd like to see https://www.npmjs.com/package/ip and `ip.isV4Format` used here, rather than a regex.